### PR TITLE
Added Animations (Makes website aesthetically pleasing)

### PR DESCRIPTION
**Pull request by mhambre**
# **Animations**
(Go to bottom to see the live page)
Since I felt that the website lacked animations I went a head and added a few. Here is the list of animations:

```
 - Nav-Bar Slide and Fade in from top
 - Footer slide and Fade in from bottom
 - About/Staff Section Fades
 - Header Content Fades
 - Rules title and Subtitle fade
 - Odd rules slide and fade in from left
 - Even rules slide and fade in from right
```

If you merge this you can delete or add animations. The CSS library is https://0-eclipse-0.github.io/Aurora_W3.CSS/w3.css. I will give @mrdnomaid and @ffxhand write access to the repository so that they can modify the classes to their liking.

Here is the live page from my forked repository so you guys can see the changes and determine whether it is worth merging: https://0-eclipse-0.github.io/AuroraWebsite/. I just feel that AuroraCommunity.co needs more than a few hovers and spins.

### CODE REVIEW

**Review by t5r7 — COMMENTED** [review]
I appreciate the effort, however this does seem a tad pointless.
I'll leave @ffxhand to decide whether to merge or not, but his opinion will likely be the same as mine.

**Comment by ghost** [comment]
Link hover animations/navbar hover animations, etc. should be `.1s` **maximum**. @0-Eclipse-0

**Comment by mhambre** [comment]
The website is practically the same layout as SSLCommunity, what SSLCommunity lacks is animations. I added the animations to make Aurora's site different from SSL and to add aesthetics. I respect your decision though as you are the creator of the site @mrdnomaid.

File: _layouts/blank.html [hunk 1]
```diff
@@ -16,6 +16,8 @@
 
   <link href="style/frameworks/bulma.css" rel="stylesheet" .>
   <link href="style/main.min.css" rel="stylesheet" />
+  <link rel="stylesheet" href="https://0-eclipse-0.github.io/Aurora_W3.CSS/w3.css">
```
Comment (ghost) [inline _layouts/blank.html:19]: Please don't include extra stylesheets.

File: _layouts/page.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ---
 <section class="hero is-medium" style="background-image: url('img/header-blur.jpg'); background-size: cover; background-position: bottom;">
   <div id="hero-head">
-    <nav class="nav">
+    <nav class="nav w3-animate-opacity w3-animate-top">
```
Comment (ghost) [inline _layouts/page.html:6]: These classes seem unnecessary. Please don't add unnecessary classes when you can easily animate via standard element selectors.

**Review by ghost — CHANGES_REQUESTED** [review]
Unnecessary new CSS sheet instead of modifying existing stylesheet, and added classes which appear unnecessary.

**Comment by mhambre** [comment]
So should I fix those @ffxhand or not?

**Comment by ghost** [comment]
Yes, otherwise I'd have approved the changes.

File: CNAME [hunk 3]
```diff
@@ -1,2 +1 @@
-auroracommunity.co
-www.auroracommunity.co
+eclip.se
```
Comment (ghost) [inline CNAME:1]: Please don't mess with our CNAME. For testing, use the default GitHub Pages URL format.

**Review by ghost — CHANGES_REQUESTED** [review]
Messing with our CNAME = bad.

File: CNAME (hunk 3 above)
Comment (mhambre) [reply CNAME:1]: Ok.

**Comment by ghost** [comment]
Closing this PR, after chatting in private we've come to the consensus that these changes would not benefit the site in any conceivable way.

**Pull request closed without merging**